kyuubi-server/src/test/scala/org/apache/kyuubi/parser/trino/KyuubiTrinoStatementSuite.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.parser.trino
+
+import org.apache.kyuubi.KyuubiFunSuite
+import org.apache.kyuubi.sql.parser.KyuubiParser
+import org.apache.kyuubi.sql.trino.TrinoGetSchemas
+
+class KyuubiTrinoStatementSuite extends KyuubiFunSuite {
```

Comment:
We can have an end-to-end test if the trino resource work done

Reply:
sure, it based on Trino jdbc-it